build.gradle

```diff
@@ -1,13 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext {
+        compose_version = '1.2.0-alpha05'
+        wear_compose_version = '1.0.0-alpha18'
+    }
     ext.kotlin_version = '1.5.31'
```

Comment:
If you update to `ext.kotlin_version` to `'1.6.10'`, I believe that should get rid of the errors you're seeing and you can add Lint back in (i.e., remove the lines that disable Lint). I was able to create a basic Wear template here that runs without disabling Lint - you can compare against this branch and see if there are any other differences:
https://github.com/barbeau/gpstest/tree/wear-template

Reply:
> If you update to `ext.kotlin_version` to `'1.6.10'`, I believe that should get rid of the errors you're seeing and you can add Lint back in (i.e., remove the lines that disable Lint). I was able to create a basic Wear template here that runs without disabling Lint - you can compare against this branch and see if there are any other differences: https://github.com/barbeau/gpstest/tree/wear-template

Thanks, but If I use version '1.6.10', the project can't pass compiling. It says "This version (1.1.0-beta02) of the Compose Compiler requires Kotlin version 1.5.31 but you appear to be using Kotlin version 1.6.10 which is not known to be compatible.  Please fix your configuration (or `suppressKotlinVersionCompatibilityCheck` but don't say I didn't warn you!).". I've created the module by creating an empty compose Activity for Wear OS.  Have we created the new module in the same way?

Reply:
Yes, I did the same thing, but  I had to manually bump the versions numbers myself and run the Gradle upgrade wizard. If you change all the version numbers I commented on together it should work. If you're still getting compilation errors after that you might need to invalidate caches and restart Android Studio.

Reply:
> bump

Thanks, I've updated the `kotlinCompilerExtensionVersion` to '1.1.0-rc02' and then updated the `ext.kotlin_version` to `1.6.10`. Now, it works well.